deployments/helms/dev/values.yaml

```diff
@@ -0,0 +1,116 @@
+affiliations: 'github_users.json'
```

Comment:
Perhaps it would be better to sort the configuration items in the configuration file, and add some comments.

Reply:
I think it should be difficult to sort it, is there any benefit to doing so? But I think the related configurations are better put together. About writing comments, maybe we need a better documentation for these configurations.

Reply:
>  we need a better documentation for these configurations.

I don't think it's needed right now. There are actually many configurations that we are not sure how to use and may need to be added gradually.

Reply:
Yes, what I means is  put the related configurations to be together.

Reply:
I think most of the content is already categorized.

Reply:
Some common configurations can be put on the front, like this:
![image](https://user-images.githubusercontent.com/5086433/115208340-05eec100-a12f-11eb-9442-35ab6cec98eb.png)

Reply:
Fixed.